src/algebra/free_non_unital_non_assoc_algebra.lean

```diff
@@ -93,7 +93,6 @@ by rw [← function.comp_app (lift R f) (of R) x, of_comp_lift]
   lift R (F ∘ (of R)) = F :=
 by rw [← lift_symm_apply, equiv.apply_symm_apply]
 
-/-- See note [partially-applied ext lemmas]. -/
 @[ext] lemma hom_ext {F₁ F₂ : non_unital_alg_hom R (free_non_unital_non_assoc_algebra R X) A}
   (h : F₁ ∘ (of R) = F₂ ∘ (of R)) : F₁ = F₂ :=
```

Comment:
My point here was that you probably don't want to express this with `∘` at all since the user will always want to unfold it - it's not that the `note` isn't relevant to the implementation, but that the reasons given by the note don't seem to apply here so there's no reason to do the implementation this way.

Reply:
Do I understand correctly then that you advocate changing the hypothesis to:
```lean
(h : ∀ x, F₁ (of R x) = F₂ (of R x))
```
in addition to dropping the note?

Reply:
Yes I think so, unless you can think of a reason not to.

Reply:
Done!